### PR TITLE
[CVP-2275] Set the serviceaccount explicitly for custom scorecard tests

### PR DESCRIPTION
**Pull request by dirgim**
Adding this change to unblock MSP efforts because the `serviceaccount` config parameter is not consumed by the scorecard correctly at this time.

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 1]
```diff
@@ -39,19 +39,23 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
   echo "CUSTOM SCORECARD TESTS DETECTED"
 
   CUSTOM_SERVICE_ACCOUNT=$(/usr/local/bin/yq r "${OPERATOR_DIR}/tests/scorecard/config.yaml" 'serviceaccount')
+  SCORECARD_SERVICE_ACCOUNT="default"
```
[the remark this answers is not in the document]
Comment (dirgim) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:43]: Ack, updated

**Review by jsztuka — APPROVED** [review]
lgmt

**Pull request merged**